### DISCUSSION TITLE
Integration with notion - Ultimate Brain

### DISCUSSION THREAD

**Discussion by edvinPL** [kalaspuff/ai-assisted-task-executor#2] [Ideas]
Hello!
I don't know if you've heard of the PARA method and BASB By Tiago Forte.
Generally, there's a IMHO better implementation called Ultimate Brain by Thomas Frank https://thomasjfrank.com/brain/
I was thinking - if this AI task executor would help by analyzing projects, areas, goals - and try to come up with small tasks and connect them to proper places and maybe try to set some deadlines?
He could also help by amassing and organizing information for given tasks (especially from already-posessed Notes database, in PARA method you start distillation by first searching for what you need). 

Basically he would execute what he can from the database and delegate the rest to us, mere humans, and make sure we do it 
(a kind-of-a-boss/self-accountability partner? Is that our future with AI, getting rushed by AI?)
